src/features/projectsV2/ProjectDetails/components/OtherInterventionInfo.tsx

```
@@ -0,0 +1,195 @@
+      plantDate={plantLocationInfo?.interventionStartDate}
+      type={plantLocationInfo?.type}
+    />,
+    plantLocationInfo?.plantedSpecies && plantLocationInfo.plantedSpecies.length > 0 && (
+      <SpeciesPlanted
+        key="speciesPlanted"
+        totalTreesCount={totalTreesCount}
+        plantedSpecies={plantLocationInfo.plantedSpecies}
+      />
+    ),
+    plantLocationInfo &&
+    plantLocationInfo.sampleInterventions && plantLocationInfo.sampleInterventions.length > 0 && (
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)